pages/requests/new/[ware].js

```diff
@@ -169,45 +169,62 @@ const NewRequest = ({ session }) => {
   }
 
   return(
-    <div className='container'>
-      <Title title={dynamicForm.name || ''} addClass='my-4' />
-      {dynamicForm.schema ? (
-        <Form
-          formData={formData}
-          onChange={e => setFormData(e.formData)}
-          onSubmit={handleSubmit}
-          schema={dynamicForm.schema}
-          uiSchema={dynamicForm.uiSchema}
-          validator={validator}
-        >
-          <StandardRequestOptions
-            defaultRequiredDate={oneWeekFromNow}
-            requestForm={requestForm}
-            updateRequestForm={updateRequestForm}
-            buttonDisabled={buttonDisabled}
-          />
-        </Form>
-      ) : (
-        <BsForm
-          onSubmit={handleSubmit}
-          id={`new-${wareID}-request-form`}
-          noValidate
-          validated={validated}
-        >
-          <BlankRequestForm updateRequestForm={updateRequestForm} />
-          <StandardRequestOptions
-            defaultRequiredDate={oneWeekFromNow}
-            requestForm={requestForm}
-            updateRequestForm={updateRequestForm}
-            buttonDisabled={buttonDisabled}
-          />
-        </BsForm>
-      )}
-    </div>
+    <>
+      {disabled &&
+        <Notice
+          addClass='mt-5'
+          alert={{
+            body: ['To proceed with making a request, please log in to your account.'],
+            title: 'Sign in required',
+            variant: 'info'
+          }}
+          dismissible={false}
+        />
```

Comment:
why move this back up here? the idea was to make the return value cleaner and extract the loading/error handling. (it's the same way on all of the other pages now too)

Reply:
it was previously just a variable, and it needed to be changed into a component. this meant it was just a component rendering a single component, which felt like it might be too much. I changed it to that setup though so it should be more similar to other pages now 👍

Reply:
gotcha. it's unexpected that it wouldn't work as is...but I also didn't test this workflow. thanks for the explanation! 🙂